webcompat/static/js/lib/bugform.js

```diff
@@ -11,6 +14,21 @@ function BugForm() {
   this.UPLOAD_LIMIT = 1024 * 1024 * 4;
   this.clickedButton = null;
   this.hasImage = null;
+  // image editor props
+  this.editImageButton = $(".js-edit-img-open-btn");
+  this.editImageModal = $("#imageEditor");
```

Comment:
Why not just `$('.modal')? Then you can get rid of the id from the element as well.